app/assets/javascripts/search.js

```diff
@@ -0,0 +1,12 @@
+$(document.body).ready(function() {
```

Comment:
Can you put this in an IIFE? 
(function() {
  'use strict';
  ....
  ....
})();